### ISSUE TITLE
[Feature Request][A11Y][TabContainer] Need a parameter to custom the aria-label text for overflow menu button

### ISSUE THREAD

**Issue by Hubery-Shen** [SAP/ui5-webcomponents#2184]
**Dear colleagues,**
I am from SuccessFactors Web Experience Shanghai team.

**Feature request**
1. Need a parameter to custom the aria-label text for overflow menu button

![FR  TabContainer](https://user-images.githubusercontent.com/63704163/92425489-034fe480-f1ba-11ea-9c6e-857fe699e431.png)

**Expected API Spec**

```
<ui5-tabcontainer class="full-width" show-overflow overflow-label="my customized overflow button label">
</ui5-tabcontainer>
```

**Comment by Hubery-Shen** [comment]
Hi @ilhan007 

We need a parameter to custom the aria-label text for overflow menu button for TabContainer component.
I read this https://github.com/SAP/ui5-webcomponents/pull/2192/files pr related to this feature request and found that a slot should be provided by us to custom the aria-label text.
We only want to customize the aria-label for the overflow menu button, not want to override the whole for overflow menu button.

Can a parameter be provided to support this feature request ?|
Thanks (edited)
